src/locale/locales/pt-BR/messages.po

```diff
@@ -2477,15 +2477,15 @@ msgstr "Mostrar ou esconder o post a que você está respondendo"
 
 #: src/view/screens/NotificationsSettings.tsx:83
 msgid "Experimental: When this preference is enabled, you'll only receive reply and quote notifications from users you follow. We'll continue to add more controls here over time."
-msgstr ""
+msgstr "Experimental: Quando essa preferência estiver habilitada, você receberá apenas notificações de resposta e citação de usuários que você segue. Continuaremos adicionando mais controles aqui ao longo do tempo"
 
 #: src/components/dialogs/MutedWords.tsx:500
 msgid "Expired"
-msgstr ""
+msgstr "Expirado "
 
 #: src/components/dialogs/MutedWords.tsx:502
 msgid "Expires {0}"
-msgstr ""
+msgstr "Expirado {0}"
```

Comment:
I think "expira" would be better since it could be used in a future context lke "expira amanhã"

Reply:
Yes, correct "Expirada"

Reply:
Yes, correct Expira